kokoro/continuous.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+export M2_HOME=/usr/local/apache-maven
```

Comment:
I've read here and there that you don't want to use `M2_HOME` going forward.

[Maven 3.5.0 release note](https://maven.apache.org/docs/3.5.0/release-notes.html)
https://stackoverflow.com/a/47712646/1701388
https://stackoverflow.com/a/17137183/1701388

Reply:
Removed M2_HOME.